open-metadata-implementation/access-services/digital-service/digital-service-api/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project.  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>digital-service</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.8-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.odpi.egeria</groupId>
+    <artifactId>digital-service-api</artifactId>
+    <name>Digital Service OMAS API</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.11.0</version>
```

Comment:
versions must not be included in the child poms. They should be added in the dependencyManagement section of the top level pom to maintain consistency. Then remove the version string here. Also ensure the current version is being used.

Reply:
addressed